Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
```

Comment:
It's not a duplicate rule because the class files could locate in any subdirectories under our java project folder.





At 2016-05-01 16:57:23, "Brendan Forster" <notifications@github.com> wrote:


In Global/Eclipse.gitignore:

> @@ -49,3 +49,15 @@ local.properties
>  
>  # Code Recommenders
>  .recommenders/
> +
> +# Java class files
> +**/*.class


This rule is captured by our Java.gitignore - the expectation here is that users will grab whatever templates they need based on their usage, so I'd like to avoid duplication wherever possible.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub